cmd/action/apply.go

```diff
@@ -47,6 +47,10 @@ func init() {
 
 // CmdApplyRun is the command used when running CLI.
 func CmdApplyRun(cmd *cobra.Command, args []string) {
+	if ApplyFlags.Web {
+		schemaCmd.PrintErrln("Open in UI server is not available in this release")
```

Comment:
```suggestion
		schemaCmd.PrintErrln("Opening the Graphical UI server is not available in this release")
```

Maybe also add instruction on how to get a release with the GUI.